.github/workflows/config.json

```diff
@@ -75,9 +75,8 @@
     },
     "eval": {
         "metric_types": [
```

Comment:
Should we also test here the new ROUGE metrics added?

Reply:
will add